### ISSUE TITLE
Worktable Clay Lump crafting not working

### ISSUE THREAD

**Issue by SuperDragonite2172** [codetaylor/pyrotech#276]
#### Issue Description

Pyrotech's recipe for 4 Clay using 8 Dirt Clumps and a bucket of water (Recipe ID: pyrotech:clay) does not want to craft using the Worktable.

#### What Happens

Placing all the required items according to JEI onto the Worktable has the progress bar show up on The One Probe, prompting for the hammer.  Using the hammer on the worktable did not fill the progress bar.  I tried a variety of Pyrotech hammers and buckets filled with water to no effect.

#### What You Expect to Happen

I expected the crafting recipe to work like it does in the standard crafting table.  The other recipes I've tried on the Wooden Worktable have worked as intended.

#### Script

N/A

#### Crash Log

N/A

#### Affected Versions

* Minecraft: 1.12.2
* Forge: 14.23.5.2847
* CraftTweaker: N/A
* Dropt: 1.16.1
* Pyrotech: 1.4.20
* Athenaeum: 1.17.4

**Comment by codetaylor** [comment]
I'm unable to reproduce this.

Are you clicking with the right mouse button?

![2020-02-14_10-17-09](https://user-images.githubusercontent.com/1240716/74556611-46dbbc00-4f13-11ea-8004-ff1c28855125.gif)

**Comment by SuperDragonite2172** [comment]
I'm certain I'm clicking the right mouse button.  I've been able to craft other items such as the Drying Rack.

**Comment by codetaylor** [comment]
Can you reproduce this in an instance using only Pyrotech and its dependencies?

**Comment by SuperDragonite2172** [comment]
I was able to resolve the issue by breaking and replacing the Worktable.  I think it may have bugged out when I picked up the table with Carry On trying to clear the table and then placing it back down (I don't know for certain).  I've since removed Carry On to prevent that issue but I didn't think that it would have caused this sort of behavior.  The recipe now works as intended with no issues.  Thank you for your time.
